gallery/src/components/NxOverflowTooltip/NxOverflowTooltipExample.scss

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+#nx-overflow-tooltip-dynamic-example {
+  .nx-p {
+    max-width: unset;
```

Comment:
Or should this be on `.nx-truncate-ellipsis` in general?

Reply:
> Or should this be on `.nx-truncate-ellipsis` in general?

Wouldn't that cause issues where there's a `max-width` set that we want to keep? That seems like an overly broad and potentially dangerous change.

Note also that in this case I'm just tweaking an example so that a visual test runs correctly.